core/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 //! Iroha — A simple, enterprise-grade decentralized ledger.
+#![feature(adt_const_params)]
```

Comment:
I support using nightly features that will reach stable soon
can we enable it project wide? probably not

Reply:
I don't find a way to enable feature project wide.